tests/unit/Certify.spec.ts

```diff
@@ -0,0 +1,175 @@
+//
+// Copyright © 2019 Province of British Columbia
```

Comment:
If we're putting in the copyright notice (which we are not doing consistently atm, so this could go either way) then please set this to 2020 :)